src/mds/MDSMap.h

```diff
@@ -416,72 +422,6 @@ class MDSMap {
 	s.insert(p->second.rank);
   } 
 
-  int get_random_up_mds() {
```

Comment:
I moved it to the place where it was used, on the principle that I don't want something visible across the entire codebase when it's only used one place.